docs/testsuite_new_language.md

```diff
@@ -0,0 +1,179 @@
+How to Add Tests in a New Language
```

Comment:
To be consistent with my comment for overview page, change this to "How to Add Tests to a New Client Language Binding"